src/handlers/deeplinks.ts

```diff
@@ -1,26 +1,25 @@
 import URL from 'url-parse';
-
 import { parseUri } from '@walletconnect/utils';
-import { initialChartExpandedStateSheetHeight } from '@/components/expanded-state/asset/ChartExpandedState';
+
 import store from '@/redux/store';
 import {
   walletConnectOnSessionRequest,
   walletConnectRemovePendingRedirect,
   walletConnectSetPendingRedirect,
 } from '@/redux/walletconnect';
+import { scheduleActionOnAssetReceived } from '@/redux/data';
+import { emitAssetRequest, emitChartsRequest } from '@/redux/explorer';
 import { fetchReverseRecordWithRetry } from '@/utils/profileUtils';
 import { defaultConfig } from '@/config/experimental';
 import { PROFILES } from '@/config/experimentalHooks';
-import { delay } from '@/helpers/utilities';
+import { delay } from '@/utils/delay';
```

Comment:
Putting this in its own file is similar: it doesn't import anything else, so no need to mock it most of the time.